src/PythonBindings/CharmCompatibility.cpp

```diff
@@ -76,15 +77,25 @@ std::string get_paths() noexcept { return "Not supported in python."; }
  * These functions are not intended to be called.
  */
 
-PUP::able::~able() {}
-
-void PUP::able::pup(PUP::er& /*p*/) {}
+PUP::able::~able() {
+  // Destructor will be called in Python, but doesn't need to do anything
+}
 
-void PUP::able::PUP_ID::setName(const char* /*name*/) {}
+void PUP::able::pup(PUP::er& /*p*/) {
+  throw std::runtime_error{
```

Comment:
probably useful to give a code comment about what this ends up being in the corresponding python.